mlog/log.go

```diff
@@ -127,6 +128,30 @@ func (l *Logger) StdLog(fields ...Field) *log.Logger {
 	return zap.NewStdLog(l.With(fields...).zap.WithOptions(getStdLogOption()))
 }
 
+// Returns a writer that can be hooked up to the output of a golang standard logger
+// anything writern will be interprated as log entries accordingly
```

Comment:
nit: adopt go-doc convention and write, `StdLogWriter returns a ...`?
typo: written
typo: interpreted